qa/workunits/ceph-helpers.sh

```diff
@@ -1286,8 +1300,14 @@ function erasure_code_plugin_exists() {
     local plugin=$1
 
     local status
+    local grepstr
+    case x`uname`x in
```

Comment:
Sane would be relative in this crazy world. 8D
I've seen too many things come up empty... The last one being readlink(1).
But you are right chances are slim.

And I'll put in a wildcard as default catch